test/suites/edgexfoundry/main_test.go

```diff
@@ -18,6 +18,28 @@ const (
 
 var start = time.Now()
 
+func TestCommon(t *testing.T) {
+	params := &utils.TestParams{
+		Snap:               "edgexfoundry",
+		App:                "core-metadata",
+		DefaultServicePort: "59881",
+		TestConfigs: utils.TestConfigs{
+			TestEnvConfig:    true,
+			TestAppConfig:    true,
+			TestGlobalConfig: true,
+			TestMixedConfig:  true,
+		},
+		TestNetworking: utils.TestNetworking{
+			TestOpenPorts:        []string{"59881"},
```

Comment:
The value isn't used. If this is redundant and always same as DefaultServicePort, then it can be changed to bool.